setup.py

```diff
@@ -85,6 +85,14 @@
             # https://cryptography.io/en/latest/api-stability/#deprecation
 
         "mock;python_version<'3.3'",
-    ]
+        ],
+    extras_require={  # It does not seem to work if being defined inside setup.cfg
+        "broker": [
+            # The broker is defined as optional dependency,
+            # so that downstream apps can opt in. The opt-in is needed, partially because
+            # most existing MSAL Python apps do not have the redirect_uri needed by broker.
+            "pymsalruntime>=0.3,<0.6;python_version>='3' and platform_system=='Windows'",
```

Comment:
>python_version>='3' [](http://example.com/codeflow?start=37&length=19)

Does this include py 3.5? We dropped support for 3.5 I believe. @MSamWils as FYI

Reply:
Good point. Previously I did not notice [pymsalruntime officially declared "requires: Python >= 3.5"](https://pypi.org/project/pymsalruntime/).

Does pymsalruntime use any Python 3.6+ syntax so that it has zero chance to run on Python <=3.5? Or is it more about our only providing build artifacts (Python wheels) for Python 3.6+? In the latter case, customers could still theoretically use msalruntime with lower version of Python 3, by building from source code.

Regardless, my next commit will align with PyMsalRuntime's declaration on PyPI.

Reply:
I misunderstood that you wanted MSAL Python to declare the lower bound as ">=3.5" (because the PyMsalRuntime itself also declares the same lower bound). Today I notice that [PyMsalRuntime is not built at Python 3.5 at all](https://pypi.org/project/pymsalruntime/#files). So, ok our next commit can bump the lower bound to ">=3.6", assuming that we do not plan to support Python<3.6 customers (which is fine, considering that [those Python versions are already end-of-support](https://endoflife.date/python)).